review of PR 2476 (#3)
l0rd/che-docs: modules/end-user-guide/examples/snip_che-table-of-supported-editors.adoc

| `che-incubator/che-code/insiders`
| This is the default IDE that loads in a new workspace when the URL parameter or `che-editor.yaml` is not used.

| link:https://github.com/che-incubator/jetbrains-editor-images[JetBrains IntelliJ IDEA Community Edition]
|
* `che-incubator/che-idea/latest`
* `che-incubator/che-idea/next`
|
* `latest` is the stable version.
* `next` is the development version.

| link:https://github.com/che-incubator/jetbrains-editor-images[JetBrains PyCharm Community Edition]
|
* `che-incubator/che-pycharm/latest`
* `che-incubator/che-pycharm/next`
|
* `latest` is the stable version.
* `next` is the development version.

| link:https://github.com/eclipse-che/che-theia[Eclipse Theia]
| `eclipse/che-theia/latest`

> | *Deprecated and will be removed in a future release.*

Author: Deprecation seems to have moved to a different release:

```suggestion
| Stable and planned for deprecation and removal in future releases.
```
